packages/ui/src/components/PropertiesModal/PropertiesModal.tsx

```diff
@@ -41,11 +66,12 @@ export const PropertiesModal: FunctionComponent<IPropertiesModalProps> = (props)
       isOpen={props.isModalOpen}
       onClose={props.onClose}
       ouiaId="BasicModal"
+      description={description}
     >
-      <p data-testid="properties-modal-description">{props.tile.description}</p>
-      <br />
-      {tabs.length == 0 && <EmptyTableState name={props.tile.name}></EmptyTableState>}
-      {tabs.length != 0 && <PropertiesTabs tabs={tabs}></PropertiesTabs>}
+      <ModalBoxBody>
+        {tabs.length == 0 && <EmptyTableState name={props.tile.name}></EmptyTableState>}
+        {tabs.length != 0 && <PropertiesTabs tab={activeTab!} tab_index={activeTabKey}></PropertiesTabs>}
```

Comment:
mmm, it seems this comparison fell through the cracks 🤔 

```suggestion
        {tabs.length === 0 ? <EmptyTableState name={props.tile.name} /> : <PropertiesTabs tab={activeTab!} tab_index={activeTabKey} />}
```

Reply:
updated, thanks!